src/apply_patches.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+if [[ "${CONTAINER_PATCH_URLS:-}" ]]; then
+  source ./logging.sh
```

Comment:
This is the bit that's making the code to failure, you don't need the ./

https://github.com/felddy/foundryvtt-docker/blob/develop/src/entrypoint.sh#L21